packages/nextcloud/generate_exports.dart

```diff
@@ -1,4 +1,3 @@
-// ignore_for_file: depend_on_referenced_packages
 import 'dart:io';
```

Comment:
I'd rather put it into tools/

Reply:
Yeah, I also don't like this. I thought about recreating this with bash but I think it is too much headache...